gettsim/tests/test_data/test_dfs_alg2.csv

```diff
@@ -21,9 +21,12 @@
 7,7,20,"FALSE","FALSE",35,500,100,75,"FALSE","FALSE",800,0,0,0,0,0,0,0,0,160,0,0,0,0,2005,1550.631375,1550.631375,506.19925,0,878.8,600,1478.8,640,133.80075,,,8,75,3,0,2400,0.7433375
 7,7,21,"FALSE","FALSE",40,500,100,75,"FALSE","FALSE",1200,0,0,0,0,0,0,0,0,240,0,0,0,0,2005,1550.631375,1550.631375,765.632625,0,878.8,600,1478.8,960,194.367375,,,8,75,3,0,2400,0.762225
 8,8,22,"TRUE","FALSE",45,300,100,20,"FALSE","FALSE",600,0,0,0,0,0,0,0,0,92,0,0,0,0,2005,415.466,415.466,415.466,0,338,200,538,508,92.534,,,10,20,1,0,600,0.771116666666667
-9,9,23,"TRUE","FALSE",30,438,80,60,"FALSE","TRUE",800,0,0,0,0,0,0,0,0,136,388,0,372,0,2019,1184,796,796,0.36,1123.64,518,1641.64,1036,240,,,8.63333333333333,60,3,0,1172,
-9,9,24,"FALSE","TRUE",8,438,80,60,"FALSE","TRUE",0,0,0,0,0,0,0,0,0,0,388,0,0,0,2019,1184,796,0,0.36,1123.64,518,1641.64,0,0,,,8.63333333333333,60,3,0,1172,
-9,9,25,"FALSE","TRUE",3,438,80,60,"FALSE","TRUE",0,0,0,0,0,0,0,0,0,0,388,0,0,0,2019,1184,796,0,0.36,1123.64,518,1641.64,0,0,,,8.63333333333333,60,3,0,1172,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,,,,,,"It has been 2019",,,,,,,,,,,,,,,,,
```

Comment:
What's this?

Reply:
There was some weird testing behavior, I didn't understand. So I excluded the test and inserted a comment. Will change this as soon we have updated the whole alg2 regrouped.

Reply:
Changed it back!